src/astro/databases/base.py

```diff
@@ -230,6 +230,20 @@ def load_file_to_table(
             chunk_size,
         )
 
+    def check_optimised_path_and_transfer(
+        self,
+        source_file: File,
+        target_table: Table,
+        chunk_size: int = DEFAULT_CHUNK_SIZE,
+        if_exists: LoadExistStrategy = "replace",
+        **kwargs,
+    ) -> bool:
+        """
+        Checks if optimised path for transfer between File location to database exists
+        and if it does, it transfers it and returns true else false.
+        """
+        raise NotImplementedError()
```

Comment:
Should we raise an exception here? we might have databases that don't have optimized paths to them like SQLite and Postgres.

Reply:
done